integrationtests/tests/fixtures/mod.rs

```diff
@@ -1210,18 +1262,11 @@ impl FederationTest {
     }
 }
 
-pub fn assert_ci<M: ModuleConsensusItem>(ci: &ConsensusItem) -> &M {
-    if let ConsensusItem::Module(mci) = ci {
-        assert_module_ci(mci)
-    } else {
-        panic!("Not a module consensus item");
-    }
-}
-
-pub fn assert_module_ci<M: ModuleConsensusItem>(mci: &PerModuleConsensusItem) -> &M {
-    debug!(
-        module_instance_id = mci.module_instance_id(),
-        "Checking module consensus item"
-    );
-    mci.as_any().downcast_ref().unwrap()
+/// Unwraps a dyn consensus item into a specific one for making assertions
+pub fn unwrap_item<M: ModuleConsensusItem>(mci: &Option<DynModuleConsensusItem>) -> &M {
```

Comment:
Might add `#[track_caller]` so when it panics, it panics with the place calling it, so it's easier to figure out which type was it.